route_following_plugin/src/route_following_plugin.cpp

```diff
@@ -149,7 +156,6 @@ namespace route_following_plugin
     {
         if (latest_maneuver_plan_.empty())
         {
-            ROS_ERROR_STREAM("A maneuver plan has not been generated");
```

Comment:
Why did you delete this line? It was not a duplicate. Please think through each change with a bit more diligence.